toolkit/themes/shared/design-system/design-tokens.json

```diff
@@ -160,31 +273,364 @@
     "white": {
       "value": "#ffffff"
     },
+    "accent": {
+      "primary": {
+        "@base": {
+          "value": {
+            "prefersContrast": "AccentColor",
+            "forcedColors": "ButtonText",
+            "brand": {
+              "default": "{brand.color.accent.@base}"
```

Comment:
I think the brand/platform colours can go here rather than assigning them to their own tokens

Reply:
Ah I see what you mean, but I'm curious if there's another reason we're bothering to assign to `--brand-color-accent`, `--platform-color-accent`, etc. @jsimplicio do we want to keep those tokens or just re-assign `--color-accent-primary` and it's variants in the brand/platform sheets?

Either way I would rather make that change as a separate patch and keep this focused on just replicating what we have in the existing files to avoid any confusion/possible scope creep.

Reply:
For now I just sorted them into the color comment blocks same as we have in central